src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -158,6 +161,16 @@
                 "ARRAY_INIT",
                 // these are covered by GenericWhitespaceCheck
                 "WILDCARD_TYPE", "GENERIC_END", "GENERIC_START").collect(Collectors.toSet()));
+        CHECKSTYLE_TOKENS_IN_CONFIG_TO_IGNORE.put("EmptyBlock", Stream.of(
+                // state of the configuration when test was made until
+                "INSTANCE_INIT", "LITERAL_DO", "LITERAL_ELSE", "LITERAL_FINALLY", "LITERAL_FOR",
+                "LITERAL_IF", "LITERAL_SWITCH", "LITERAL_SYNCHRONIZED", "LITERAL_TRY",
+                "LITERAL_WHILE", "STATIC_INIT")
+                .collect(Collectors.toSet()));
+        CHECKSTYLE_TOKENS_IN_CONFIG_TO_IGNORE.put("RightCurly", Stream.of(
+                // state of the configuration when test was made until
+                "LITERAL_CATCH", "LITERAL_ELSE", "LITERAL_FINALLY", "LITERAL_IF", "LITERAL_TRY")
```

Comment:
> default of RightCurly is same.
I think we should be same 
Am I miss smth ?

Let's take `LITERAL_IF` and `LITERAL_ELSE` as an example from the ones missing.
If we switched them to `same` then we would have to write if/else like this:
````
    if (condition) {
       ...
    } else {
       ...
    }
````
while, as of right now, most of our if/else is written as:
````
    if (condition) {
       ...
    }
    else {
       ...
    }
````
Violations for `alone` are 9. Violations for `same` are 904.
We like writting our curlies alone on the line.